site/docs/v1/tech/apis/form-fields/_form-field-type.adoc

```diff
@@ -13,4 +13,8 @@ When using the `date` type, the only supported data format is ISO-8601 or `YYY-M
 +
 When using the `consent` type, the field [field]#consentId# must be included in the request.
 +
-Defaults to `string`, unless `user.birthDate`, then `date`, or if `user.email` then `email`.
\ No newline at end of file
+Defaults can change depending on the type of `key`. The defaults are:
++
+* `bool` if key is `user.twoFactorEnabled`
+* `date` if key is `user.birthDate`
+* `email` if key is `user.email`
```

Comment:
Is there a fall through default, or is it required if not one of these values?

Reply:
Completed.